### DISCUSSION TITLE
AMS MSc Recuirements

### DISCUSSION THREAD

**Discussion by edzob** [JRBliekendaal/master-thesis#10] [General]
Had je ergens vanuit AMS al hun officiele recuirements list gehad?

met oa max 1 pagina executive summary, APA reference style, in actieve vorm geschreven (etc)

**Comment by JRBliekendaal** [comment]
Yup.. heb die gehad van Engelina en Danny... Had ze vorig jaar al. Is trouwens niet heel anders dan de requirements welke ik had van het NOVI.

**Comment by JRBliekendaal** [comment]
If you want to see if something changed (I do not expect that it is)

[documents](https://github.com/JRBliekendaal/master-thesis-administration/tree/main/ams%20specific)

**Reply by edzob** [reply]
chill .
Die matrix hebben wij volgens mij nooit gehad. Heel fijn dat ze die nu (wel) uitdelen.
Alle energie kan dan in het onderzoek zelf gaan zitten ipv deze randzaken. ;)
